src/instrumenter/instrument.ts

```diff
@@ -523,8 +562,9 @@ export function insertAnnotations(annotations: PropertyMetaData[], ctx: Transpil
     const checkStmts: Statement[] = predicates.map(([annotation, predicate], i) => {
         const event = getAssertionFailedEvent(factory, contract);
         const dbgInfo = debugInfos[i];
-        const emitStmt = dbgInfo !== undefined ? dbgInfo[1] : undefined;
-        return emitAssert(ctx, predicate, annotation, event, emitStmt);
+        const emitStmt = dbgInfo !== undefined ? dbgInfo[2] : undefined;
+        const encoding = dbgInfo !== undefined ? dbgInfo[1] : undefined;
```

Comment:
After you remove the separate assignment in getDebugInfo remove the extra const here